src/MainView.mc

```diff
@@ -91,16 +86,35 @@ class MainView extends WatchUi.View {
   function drawProgress(dc, value, max, codeColor) {
     dc.setPenWidth(dc.getHeight() / 40);
     dc.setColor(codeColor, Graphics.COLOR_TRANSPARENT);
+    var subscreen = Subscreen.getSubscreen();
     if (screen_shape_== System.SCREEN_SHAPE_ROUND) {
       // Available from 3.2.0
       if (dc has :setAntiAlias) {
         dc.setAntiAlias(true);
       }
-      dc.drawArc(dc.getWidth() / 2, dc.getHeight() / 2, (dc.getWidth() / 2) - 2, Graphics.ARC_COUNTER_CLOCKWISE, 90, ((value * 360) / max) + 90);
+      dc.drawArc(
+        dc.getWidth() / 2,
+        dc.getHeight() / 2,
+        (dc.getWidth() / 2) - 2,
+        Graphics.ARC_COUNTER_CLOCKWISE,
+        90, ((value * 360) / max) + 90
+      );
       // Available from 3.2.0
       if (dc has :setAntiAlias) {
         dc.setAntiAlias(false);
       }
+    } else if (subscreen != null) {
+      dc.setPenWidth(subscreen.width / 2);
+      dc.drawArc(
+        subscreen.x + subscreen.width / 2,
+        subscreen.y + subscreen.height / 2,
+        // I don't understand how the radius parameter works
+        // Apparently a quarter width works to get a complete circle
+        subscreen.width / 4,
+        Graphics.ARC_COUNTER_CLOCKWISE,
+        90, ((value * 360) / max) + 90
```

Comment:
Nice rendering of the countdown in the subscreen.

![image](https://github.com/ch1bo/garmin-otp-authenticator/assets/2621189/8830b414-054a-42ab-873d-6c3ce7aa0de9)

I wonder whether we need the time below the code? It seems we have quite limited space on the instinct2 and 2s

Reply:
What do you thing of this?
<img width="333" alt="Screenshot 2023-07-20 at 10 55 40" src="https://github.com/ch1bo/garmin-otp-authenticator/assets/11903688/f7efee65-5668-4aa8-a693-07f925999429">

Reply:
And thanks for the review btw.

Also, I had an idea for an alternative service selection UI (because I have 13 services now and it's getting tedious to scroll through them):
- User is presented with letter selection for first letter of service (for example a choice between "A", "M", and "G") if they have services starting with those letters.
- User selects letter
- Depending on the number of services that started with that letter, show the services in a list or show another letter selection screen for the second letter. Because if there's only like three services left it'd be better to just show them immediately.

Reply:
Layout looks good. I left a note that we should work towards device specific layout / view classes or so to make the rendering code less conditional.

That service selection idea is definitely another PR in itself :)